client/src/components/DataEntry/BoardTable.tsx

```diff
@@ -86,6 +107,18 @@ const BoardTable: React.FC<IProps> = ({ boardName, ballots, url }: IProps) => {
     b => b.status === BallotStatus.NOT_AUDITED
   )
 
+  const getTotalAudited = ballots.filter(
```

Comment:
nit: maybe just `totalAudited` since this is just an int, not a function to "get" something